src/views/projects.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+
+    <meta charset="en">
+
+<head>
+    <h1>Progress Oriented Projects</h1>
+    <p1>
+        Devlink<br>
+        -a place where developers can find eachother and collaborate<br>
+        -Tools: React, Node JS, MongoDB<br>
+    </p1>
```

Comment:
We should group relevant information into `div`s, that way they can be repeated easily. Also, because the information is different, let's add some semantic HTML to this!

```html
<div>
    <h2>Devlink</h2>
    <p>A place where developers can find each other and collaborate</p>
    <div><span>React</span><span>NodeJS</span><span>MongoDB</span></div>
</div>
```

You can repeat this for the other HTML as well.